src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -550,7 +550,7 @@ def dotnet(
         if target_options is None:
             target_options = []
         target_options = [
-            "--target_path={setup_dir}/" f"{target_harness}"
+            "--target_path={setup_dir}/" + "{target_harness}"
```

Comment:
Opportunistic fix, decided this was too brittle (works for constants, but not variables).